apps/operator/src/scanner/scanners/persistent-volume.scanner.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@nestjs/common';
+import { BaseResourceScanner } from '../base.scanner';
+import * as k8s from '@kubernetes/client-node';
+import { KubeService } from '../../kube/kube.service';
+import { ConfigService } from '../../config/config.service';
+import { CleanupResult } from '../../types';
+import { enrichKubernetesObject } from '../../utils/kube';
+
+@Injectable()
+export class PersistentVolumeScanner extends BaseResourceScanner<k8s.V1PersistentVolume> {
+
+  private usedPersistentVolumes: Set<string>;
+
+  constructor(private readonly kubeService: KubeService, config: ConfigService) {
+    super(config);
+  }
+
+  async scan(): Promise<k8s.V1PersistentVolume[]> {
+    try {
+      const response = await this.kubeService.coreApi.listPersistentVolume();
+      return response.items.map((pv) => enrichKubernetesObject(pv, 'PersistentVolume') as k8s.V1PersistentVolume);
+    } catch (error) {
+      this.logger.error(`Failed to scan PersistentVolume: ${error.message}`);
+      throw error;
+    }
+  }
+
+  async isOrphaned(pv: k8s.V1PersistentVolume): Promise<boolean> {
+    try {
+      return !(pv.status?.phase === 'Bound');
+      
+    } catch (error) {
+      this.logger.error(`Failed to check PersistentVolume ${pv.metadata.name}: ${error.message}`);
+      throw error;
+    }
```

Comment:
Looks good, I don't think we need the try, catch here.
In addition I would use a simpler condition:
```suggestion
return pv.status?.phase !== 'Bound';
```

Reply:
Done.